src/core/dynamic_batch_scheduler.cc

```diff
@@ -72,6 +72,18 @@ DynamicBatchScheduler::DynamicBatchScheduler(
   // caching enabled for model to utilize response cache.
   response_cache_enabled_ =
       (model_->Server()->ResponseCacheEnabled() && response_cache_enable);
+#ifdef TRITON_ENABLE_METRICS
+  // Initialize metric reporter for cache statistics if cache enabled
+  if (response_cache_enabled_) {
+    // The cache isn't tied to any specific model instance or GPU device,
+    // so we use device=-2 to indicate caching for now. -1 is currently
+    // used for CPU
+    const int id = -2;
```

Comment:
Instead of using the magic numbers here and various different place for device id as constant literal, may be define this in constants.h and use that here?

Reply:
Would it be OK to make this change in a follow-up PR? This PR is already pretty large on its own

Reply:
Sure!